client/settings/fraud-protection/advanced-settings/cards/purchase-price-threshold.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * External dependencies
+ */
+import React, { useContext, useEffect, useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import AmountInput from 'wcpay/components/amount-input';
+
+/**
+ * Internal dependencies
+ */
+import FraudProtectionRuleCard from '../rule-card';
+import FraudProtectionRuleToggle from '../rule-toggle';
+import FraudProtectionRuleDescription from '../rule-description';
+import FraudProtectionRuleCardNotice from '../rule-card-notice';
+import FraudPreventionSettingsContext from '../context';
+
+const PurchasePriceThresholdCustomForm = ( { setting } ) => {
+	const {
+		advancedFraudProtectionSettings,
+		setAdvancedFraudProtectionSettings,
+	} = useContext( FraudPreventionSettingsContext );
+
+	const [ minAmount, setMinAmount ] = useState(
+		advancedFraudProtectionSettings[ setting ].min_amount ?? ''
+	);
+	const [ maxAmount, setMaxAmount ] = useState(
+		advancedFraudProtectionSettings[ setting ].max_amount ?? ''
+	);
+
+	useEffect( () => {
+		advancedFraudProtectionSettings[ setting ].min_amount =
+			0 === parseFloat( minAmount ) ? null : parseFloat( minAmount );
+		advancedFraudProtectionSettings[ setting ].max_amount =
+			0 === parseFloat( maxAmount ) ? null : parseFloat( maxAmount );
+		setAdvancedFraudProtectionSettings( advancedFraudProtectionSettings );
+	}, [
+		setting,
+		minAmount,
+		maxAmount,
+		advancedFraudProtectionSettings,
+		setAdvancedFraudProtectionSettings,
+	] );
+
+	const areInputsEmpty =
+		! parseFloat( minAmount ) && ! parseFloat( maxAmount );
+	const isMinGreaterThanMax =
+		parseFloat( minAmount ) > parseFloat( maxAmount );
+
+	return (
+		<div className="fraud-protection-rule-toggle-children-container">
+			<strong>Limits</strong>
+			<div className="fraud-protection-rule-toggle-children-horizontal-form">
+				<div className="fraud-protection-rule-toggle-children-vertical-form">
+					<label htmlFor="fraud-protection-purchase-price-minimum">
+						{ __(
+							'Minimum purchase price',
+							'woocommerce-payments'
+						) }
+					</label>
+					<AmountInput
```

Comment:
> Is this the expected behavior or should the initial value be an empty string?
I believe it's the expected behavior, because the help text says "Leave blank for no limit", and it's not blank. You can also save 0 - 0 as the min max amounts, because it'll pass the checks. Treating 0 as "blank" doesn't seem right to me. But being able to save it with an error message shown is not right.